src/native/managed/cdacreader/tests/ExecutionManager/ExecutionManagerTestBuilder.cs

```diff
@@ -13,6 +14,7 @@ internal class ExecutionManagerTestBuilder
 {
     public const ulong ExecutionManagerCodeRangeMapAddress = 0x000a_fff0;
 
+    const bool UseFunclets = true;
```

Comment:
Is this needed? A `const bool` seems odd to me. If this is important, let's add a comment what this is trying to capture.